javascript/builtins/Intl.json

```diff
@@ -708,14 +708,21 @@
                   "version_added": false
                 },
                 "chrome": {
-                  "version_added": "55",
-                  "flag": {
-                    "type": "compile_flag",
-                    "name": "--datetime-format-to-parts"
-                  }
+                  {
+                    "version_added": "57"
+                  },
+                  {
+                    "version_added": "55",
+                    "version_removed": "60",
+                    "flag": {
+                      "type": "preference",
```

Comment:
Why did you change this from compile_flag to preference? I think it was a compile_flag.

Reply:
No, it's not a compile flag but a runtime flag that can be specified like this: 

chrome --js-flags="--datetime-format-to-parts"  

In v8 standalone, 

d8 --datetime-format-to-parts 

If there were a separate enum for 'runtime-flag', I'd use it, but there are only two, 'preference' and 'compile-time flag'. So, I have to choose the former.

Reply:
Thanks for clarifying, I agree we need runtime flags as an option in the schema. I've opened https://github.com/mdn/browser-compat-data/pull/615 for this.